File: specs/master/query_dsl/geo/bounding_box/geo_bounding_box_query.ts

/**namespace:QueryDsl.Geo.BoundingBox */
/**custom_serialization*/
interface geo_bounding_box_query {
	BoundingBox: bounding_box;
	type: GeoExecution;
	coerce: boolean;
	ignore_malformed: boolean;
	validation_method: GeoValidationMethod;
}

File: output/csharp/QueryDsl/Geo/BoundingBox/GeoBoundingBoxQuery.cs
using Nest.QueryDsl;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.QueryDsl {

	public class GeoBoundingBoxQuery  {
		
		[DataMember(Name="bounding_box")]
		public BoundingBox BoundingBox { get; set; }


		[DataMember(Name="type")]
		public GeoExecution Type { get; set; }


		[DataMember(Name="validation_method")]
		public GeoValidationMethod ValidationMethod { get; set; }

	}
}
